test/TestCommandLineInvocationSameAsConfig.py

```diff
@@ -1,82 +1,72 @@
-import unittest
-from subprocess import Popen, PIPE
-import os
-import shutil
-import yaml
+import pytest
 
+from ansiblelint import cli
 
-class TestCommandLineInvocationSameAsConfig(unittest.TestCase):
-    def setUp(self):
-        if os.path.exists(".sandbox"):
-            shutil.rmtree(".sandbox")
 
-        os.makedirs(".sandbox/subdir")
+@pytest.fixture
+def base_arguments():
+    return ['../test/skiptasks.yml']
 
-    def run_ansible_lint(self, args=False, config=None):
-        command = "cd .sandbox; ../bin/ansible-lint ../test/skiptasks.yml"
-        if args:
-            command += " " + args
 
-        if config:
-            with open(".sandbox/.ansible-lint", "w") as outfile:
-                yaml.dump(config, outfile, default_flow_style=False)
+@pytest.mark.parametrize('args,config', [
+                         (["-p"], "test/fixtures/parseable.yml"),
```

Comment:
**_FIXME:_**

In the future, I'd prefer putting the fixtures in the test module and writing them to disk into a tmp dir as necessary. Like https://github.com/ansible/ansible-lint/blob/master/test/TestIncludeMissingFileRule.py.

But since there's already so much effort put into this task, I won't request it to be done in this PR. Hopefully, somebody can refactor this separately later.